boa/src/builtins/regexp/mod.rs

```diff
@@ -120,6 +120,7 @@ impl BuiltIn for RegExp {
         .method(Self::test, "test", 1)
         .method(Self::exec, "exec", 1)
         .method(Self::to_string, "toString", 0)
+        .method(Self::search, "search", 1)
```

Comment:
This needs to be the `search` Symbol instead, should be
```suggestion
        .method(Self::search, (WellKnownSymbols::search(), "[Symbol.search]"), 1)
```

Reply:
Fixed.